### PR TITLE
Node Consistency 61-70

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose

The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**62) NODE: Number From Feet and Inches**
CHANGES:
Description: A length defined as feet and inches is turned into a number representation in feet.
Output: double
Output Description: Number representation in feet

**63) NODE: Math.Sqrt**
CHANGES:
Output: double

**64) NODE: Math.Sinh**
CHANGES:
Output: double

**65) NODE: Math.Sin**
CHANGES:
Output: double

**66) NODE: Math.Log10**
CHANGES:
Output: double

**67) NODE: Math..Log(number,logBase)**
CHANGES:
Output: double

**68) NODE: Math.Log**
CHANGES:
Output: double

**69) NODE: Math.Exp**
CHANGES:
Output: double

**70) NODE: Math.EvaluateFormula**
CHANGES:
Output: double
Output description: type: var[]..[] (result of the formula calculation)

image

### Reviewers
@SHKnudsen
@StudioLE
@MarkThorley
@anrova

### FYIs
-No FYI

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.8.0.2015")]
+[assembly: AssemblyFileVersion("2.9.0.2505")]
```
Comment (SHKnudsen) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: we shouldnt commit this file

File: src/Libraries/UnitsUI/UnitsUI.cs [hunk 2]
```diff
@@ -227,7 +227,7 @@ public LengthFromString():base()
         {
             Measure = Length.FromDouble(0.0, LengthUnit.FractionalFoot);
 
-            OutPorts.Add(new PortModel(PortType.Output, this, new PortData(Resources.LengthFromStringPortDataLengthToolTip, Resources.LengthFromStringPortDataLengthToolTip)));
+            OutPorts.Add(new PortModel(PortType.Output, this, new PortData("feet", Resources.LengthFromStringPortDataLengthToolTip)));
```
Comment (SHKnudsen) [inline src/Libraries/UnitsUI/UnitsUI.cs]: hm, `feet` should be in Resources. Why are we removing the `Resources.LengthFromStringPortDataLengthToolTip`?

Comment (SHKnudsen) [reply src/Libraries/UnitsUI/UnitsUI.cs]: ah its using `LengthFromStringPortDataLengthToolTip` as the name and the tooltip. Anyways the name of the port should be in a resource file as well

**Comment by SHKnudsen** [comment]
Can we add an image to the description

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (martinstacey) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: Reverted in PR 
b8a9a21

File: src/Libraries/UnitsUI/UnitsUI.cs (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/UnitsUI/UnitsUI.cs]: Tooltips are only used on descriptions, not on names in other ports.

Comment (martinstacey) [reply src/Libraries/UnitsUI/UnitsUI.cs]: Tooltips are only used on descriptions not on names in other ports.

**Pull request merged**